### PR TITLE
add check if the submitted filename is a directory

### PR DESCRIPTION
**Pull request by alebaffa**
I tried to add some tests, but I didn't know if to add a new one into_ paths/paths_test.go_ (so basically to testing the function _IsDir()_), or create a new _cmd/submit_test.go_. 
Let me know if you need a test.

### CODE REVIEW

File: cmd/submit.go [hunk 1]
```diff
@@ -60,6 +61,10 @@ func Submit(ctx *cli.Context) {
 			log.Fatal("You cannot submit the README as a solution.")
 		}
 
+		if paths.IsDir(filename) {
+			log.Fatal("You cannot submit a directory. Submit only the solution file.")
```
Comment (kytrinyx) [inline cmd/submit.go]: You can submit more than one file.

How about:

```
Please specify each file that should be submitted, e.g. `exercism submit file1 file2 file3`.
```

**Comment by kytrinyx** [comment]
No, don't worry about a test. Eventually we'll be adding tests for the commands, but it's going to take some thought to get it right.

**Comment by alebaffa** [comment]
Just for my curiosity: what are the considerations you are making for the tests for the commands? (don't know if it's the right place to ask it.. :P )

**Comment by Tonkpils** [comment]
> what are the considerations you are making for the tests for the commands?

We are using a 3rd party library to handle the CLI interface. In order to test the commands we have to mock the argument passed into the command functions which would require a change of signature of all the functions to pass an interface of our own. However, the CLI package takes a specific struct which doesn't allow for this kind of change. 

Unless that's changed in the CLI package, that's the one thing stopping us from writing tests around the commands.

**Comment by kytrinyx** [comment]
This looks good to me. Would you mind squashing the commits, please?

**Comment by alebaffa** [comment]
Voila :)

**Comment by kytrinyx** [comment]
Lovely, thank you!

**Comment by mhelmetag** [comment]
Fixes #302

**Pull request merged**